src/libs/Navigation/OnyxTabNavigator.tsx

```diff
@@ -34,6 +37,7 @@ export const TopTab = createMaterialTopTabNavigator();
 // except ID is now required, and it gets a `selectedTab` from Onyx
 function OnyxTabNavigator({id, selectedTab = '', children, onTabSelected = () => {}, screenListeners, ...rest}: OnyxTabNavigatorProps) {
     return (
+        // @ts-expect-error: Types of property 'tabBar' are incompatible.
```

Comment:
Why are we creating an exception here? Shouldn't we be fixing the actual issue?

Reply:
Ok. I'll fix it.

Reply:
You could try assertion here, instead of ts-expect-error
```ts
tabBar={tabBar as (props: MaterialTopTabBarProps) => React.ReactNode}
```

Reply:
Or better, fix `TabSelectorProps` type 😄

Reply:
> You could try assertion here, instead of ts-expect-error

You mean I should use it inside `<TopTab.Navigator`?

Reply:
Yes

Reply:
You didn't destruct tabBar from props 😅

Reply:
I think TabSelectorProps is broken, try this:

```ts
type TabSelectorProps = MaterialTopTabBarProps & {
    /* Callback fired when tab is pressed */
    onTabPress?: (name: string) => void;
};
```

Reply:
With no changes in TabNavigator